pulp_rpm/app/viewsets.py

```diff
@@ -296,14 +592,88 @@ def create(self, request):
         return OperationPostponedResponse(result, request)
 
 
-class RpmDistributionViewSet(DistributionViewSet):
+class RpmDistributionViewSet(DistributionViewSet, RolesMixin):
     """
     ViewSet for RPM Distributions.
     """
 
     endpoint_name = "rpm"
     queryset = RpmDistribution.objects.all()
     serializer_class = RpmDistributionSerializer
+    queryset_filtering_required_permission = "rpm.view_rpmdistribution"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "my_permissions"],
+                "principal": ["authenticated"],
+                "effect": "allow",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:rpm.view_rpmdistribution",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition_expression": [
+                    "has_publication_param_model_or_obj_perms:rpm.view_rpmpublication"
+                    " or has_repo_attr_model_or_obj_perms:rpm.view_rpmrepository",
```

Comment:
why there is `or` here?

Reply:
mistake, whole line does not need to be there